ooiservices/app/celeryconfig.py

```diff
@@ -21,4 +21,9 @@
         'schedule': crontab(hour='*/8'),
         'args': (),
         },
+    'get-cam-images-every': {
+        'task': 'tasks.compile_cam_images',
+        'schedule': crontab(hour='*/12'),
```

Comment:
@birdage we've updated this code overall, but you'll need to update this as well to be:

crontab(minute=0, hour='*/12'),

for some reason celery doesn't like omitting the minutes, even if it's 0